R/crosstalk.R

```diff
@@ -333,7 +333,7 @@ SharedData <- R6Class(
 
         # .updateSelection needs logical array of length nrow(data)
         # .selectionCV$sendUpdate needs character array of keys
-        isolate({
+        shiny::isolate({
```

Comment:
With shims for all the other shiny methods, should this be something like...

```r
isolate <- function(env) {
  if (shinyInstalled()) {
    shiny::isolate(env)
  } else {
    force(env)
  }
}
```

Reply:
In this case, I want it to fail. (Though maybe I should be more explicit about making things fail if they really do require shiny?)

Reply:
I have always been confused when I run into a `::` package not found error.  

I prefer the explicit failure... ``stop("`shiny` must be installed ...")``

Reply:
I didn't see the error message above. The original error message using `shiny::isolate` looks good to me.